src/Components/Timer/Timer.module.scss

```diff
@@ -0,0 +1,84 @@
+@import '../../style/variables.scss';
+
+/* Structural Styling */
+.countdownTimer {
+    display: flex;
+    flex-direction: column;
+    margin: auto;
+    width: 300px;
+}
+
+.timerControls {
+    width: 300px;
+    display: flex;
+    justify-content: space-around;
+}
+
+.timerControlButton {
+    border-radius: 50%;
+    width: 50px;
+    height: 50px;
+    align-items: center;
+    background-color: $white;
+    color: $darkBlue;
+    font-family: $primaryFont;
+    font-weight: 700;
+}
+
+.timerControlButton:hover {
```

Comment:
Yay hover styling! I'm all about that :)

Reply:
I saw in one of your PRs the `& hover` trick and meant to update it!! This is a note to self to add that in later.

Reply:
Sweet!!

Reply:
Ya that nesting is a really great feature for scss!